package.json

```diff
@@ -230,12 +230,14 @@
     "js-yaml": "^4.1.0",
     "jsonc-eslint-parser": "~2.4.0",
     "jsonpath-plus": "^8.0.0",
+    "lighthouse": "9.6.8",
```

Comment:
why frozen versions for the new deps?

Reply:
newer versions don't seem to support CommonJS https://github.com/abhinaba-ghosh/playwright-lighthouse/issues/51